text - problem scrolling

**Describe the bug**
Every time I try to scroll until the end of the library section or a playlist, there a big extra space at the end, i checked if there's something missing that i am not seeing in that space (like other playlists or songs from them) but there is not, it's extra with nothing inside.

**To Reproduce**
Steps to reproduce the behavior:
1. Scroll through the left panel where you can find your music / or a playlist
2. Scroll all the way down

**Expected behavior**
After the list ends, it should show something at the end with the correct spacing.

**Screenshots**
![image](https://github.com/spicetify/spicetify-themes/assets/86584280/cd887ade-e322-45c2-998a-324f219c5a6c)

**Logs**

```console
gue1-spclient.spotify.com/remote-config-resolver/v3/configuration:1     Failed to load resource: the server responded with a status of 404 ()
vendor~xpui.js:29 provider:transport resolve error T Object
f @ vendor~xpui.js:29
index.html:1 Uncaught (in promise) T
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/vendor~xpui.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/1517.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/home-hpto.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/1400.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/5036.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui-desktop-modals.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/home-v2.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
vendor~xpui.js:29 Error: Invalid playlist response!
    at Uo (xpui.js:1:1860795)
    at Ho.getPlaylist (xpui.js:1:1862400)
    at async Ho.getMetadata (xpui.js:1:1862626)
```

**Specifics (please complete the following information):**
 - OS: Windows 11 Pro 22H2 22621.2283
 - Spotify version: 1.2.17.834.g26ee1129
 - Spicetify version: Spicetify v2.23.2
 - Theme name: text (color scheme Spotify)

--- a/text/README.md
+++ b/text/README.md
@@ -100,10 +100,12 @@ created by [darkthemer](https://github.com/darkthemer/)
 
 ### Notes
 
--   **IMPORTANT:** Add the following to your `config-xpui.ini` file. Details as to why are explained [here](https://github.com/JulienMaille/spicetify-dynamic-theme#important).
+-   **IMPORTANT:** Add the following to your `config-xpui.ini` file. Details as to why are explained [here](https://github.com/JulienMaille/spicetify-dynamic-theme#important). Run `spicetify apply` after adding these lines.
 
 ```ini
 [Patch]
+xpui.js_find_0880 = COLLAPSED\?64:32
+xpui.js_repl_0880 = COLLAPSED?32:32
 xpui.js_find_8008 = ,(\w+=)56,
 xpui.js_repl_8008 = ,${1}32,
 ```

